inc/rest-api.php

```diff
@@ -58,7 +58,7 @@ function register_rest_routes() {
  * @return \WP_REST_Response REST API response.
  */
 function rest_profile_search( \WP_REST_Request $request ) {
-	$posts    = get_posts(
+	$posts    = get_posts( // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.get_posts_get_posts
```

Comment:
Is it supposed to say get_posts_get_posts ?

Reply:
Yes